entproto/cmd/protoc-gen-entgrpc/template/enums.tmpl

```diff
@@ -0,0 +1,23 @@
+{{ define "enums" }}
+    {{- $root := . -}}
+    {{ range .FieldMap.Enums }}
+        {{- $enumType := .PbFieldDescriptor.GetEnumType -}}
+        {{- $enumName := print $root.EntType.Name "_" $enumType.GetName -}}
+        {{- $pbEnumIdent := $root.File.GoImportPath.Ident $enumName   -}}
+        {{- $entLcase := snake $root.EntType.Name -}}
```

Comment:
```suggestion
    {{ range .FieldMap.Enums }}
        {{- $enumType := .PbFieldDescriptor.GetEnumType -}}
        {{- $enumName := print $.EntType.Name "_" $enumType.GetName -}}
        {{- $pbEnumIdent := $.File.GoImportPath.Ident $enumName   -}}
        {{- $entLcase := snake $.EntType.Name -}}
```

Reply:
BTW, you don't need all these `-` in the top-level declaration if you're running `go/format`/`imports` at the end.